<launch>

    <!-- Start calibration server -->
    <node pkg="min_variance_calibration" type="RunCalibrationServer"
      name="calibration_server" output="screen">
      <!-- TODO: Implement this -->
        <!-- <param name="root_link" value="/base_link"/>
        <param name="tip_link" value="/fisheye"/> -->
    </node>

    <!-- Run calibration client -->
    <node pkg="min_variance_calibration" type="calibration_bridge.py"
      name="calibration_bridge" output="screen" required="True">
        <param name="initial_param_yaml" type="string"
          value="$(find min_variance_calibration)/examples/initial_params.yaml"/>
        <param name="data_bagfile" type="string"
          value="$(find min_variance_calibration)/examples/calibration_data.bag"/>

        <param name="rho_start" type="double" value="10"/>
        <param name="rho_end" type="double" value="1e-6"/>
        <param name="max_f_evals" type="double" value="10000"/>
    </node>

</launch>
